controls/bsi_sys_1_6.yml

```diff
@@ -194,17 +194,27 @@ controls:
     levels:
     - standard
     description: >-
-      The sources of images that have been classified as trusted and SHOULD be adequately
-      documented along with the corresponding reasons. In addition, the process of how images or
+      (1) The sources of images that have been classified as trusted and SHOULD be adequately
+      documented along with the corresponding reasons. (2) In addition, the process of how images or
       the software components contained in an image are obtained from trusted sources and
       eventually deployed to a productive environment SHOULD be adequately documented.
-      Images used SHOULD have metadata that makes their function and history traceable. Digital
+      (3) Images used SHOULD have metadata that makes their function and history traceable. (4) Digital
       signatures SHOULD secure each image against modification.
     notes: >-
-      ToDo
-    status: manual
-    #rules:
-
+      Section 1: This requirement must be implemented organizationally.
```

Comment:
How about adding these rules?
- `approved_registries`
- `ocp_allowed_registries`
- `ocp_allowed_registries_for_import`

Reply:
approved_registries is a AWS EKS Rule, I would not use it.

Reply:
I will add the other two rules

Reply:
New version pushed